docs/installation.md

```diff
@@ -9,21 +9,20 @@
 
 #Linux/Mac Automatic installation
 ### Easy installation
-1. Run setup.sh -e
-    This will create the needed encrypted.so file
-2. Run setup.sh -i
-    This will install the bot and all stuff that is needed to run it (follow the guide)
-3. Run setup.sh -c
-    This will make the config file needed, only basic stuff is changed here. If you want to edit more edit this file after: config/config.json
-4. Run run.sh
-    This will run the bot and will start leveling your pokemon go account.
+1. Clone the git: `git clone https://github.com/PokemonGoF/PokemonGo-Bot`
+2. Go into the new directory: `cd PokemonGo-Bot`
+3. Run `./setup.sh -i`  
+    This will install the bot and all stuff that is needed to run it (follow the steps in this process)
+4. Run `./run.sh`  
+    After you are done following it this will start your bot.
 
 ### To update
 1. Stop the bot if it's running. (use control + c twice to stop it)
-2. Run setup.sh -r
+2. Run `./setup.sh -r`  
     This will reset and makes sure you have no changes made to any code since it will overide it
-3. Run setup.sh -u
-    This will run git pull and will update to the new git update.
+3. Rerun the bot `./run.sh`
+
+note: we do not support windows at this time
```

Comment:
yes but I have a new windows instruction coming soon that explains everything until than we don't support it (I am working on this right now hopefully it will be done today.)